app/views/schools/contacts/_form.html.erb

```diff
@@ -4,7 +4,7 @@
 
 
 <% if f.object.user.nil? %>
-  <%= f.input :staff_role_id, label: t('schools.contacts.form.labels.role'), collection: StaffRole.order(:title) %>
+  <%= f.input :staff_role_id, label: t('schools.contacts.form.labels.role'), collection: StaffRole.order(:title).map { |staff_role| [staff_role.translated_title, staff_role.id] } %>
```

Comment:
this should be `StaffRole.all.map { |staff_role| [staff_role.translated_title, staff_role.id] }.sort` - will fix